src/Sylius/Component/Resource/spec/Metadata/MetadataSpec.php

```diff
@@ -32,10 +29,10 @@ function let()
                 'driver' => 'doctrine/orm',
                 'templates' => 'SyliusProductBundle:Product',
                 'classes' => [
-                    'model' => Product::class,
+                    'model' => 'Sylius\Component\Product\Model\Product',
```

Comment:
IMHO it is still coupled, we should avoid any reference to existing components. Can we use `\Example\Resource` instead here and something similar everything else?